x-team/xp-ui
# Add SkillsSelector component

**Release Type:** *Non-breaking changes*

Fixes https://x-team-internal.atlassian.net/browse/XP-3570

## Description

Add SkillsSelector component

## Checklist


## Steps to Test or Reproduce

- Check the component at http://localhost:9001/?path=/story/core-components-form-components-skillsselector--basic-usage

Please confirm expected behaviour from https://x-team-internal.atlassian.net/browse/XP-3570 AC.

## Impacted Areas in Application

New `SkillsSelector` component added.

## Screenshots

![2020-04-07 13 56 40](https://user-images.githubusercontent.com/131859/78698113-2e1ad180-78d8-11ea-8597-922d0ddcb9e8.gif)
